archivos-de-la-sociedad-espanola-de-oftalmologia.csl

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-ES">
+  <info>
+    <title>Archivos de la Sociedad Espanola de Oftalmologia (Spanish)</title>
```

Comment:
The title should include the diacritics, i.e. "Archivos de la Sociedad Española de Oftalmología (Spanish)"  (http://www.journals.elsevier.com/archivos-de-la-sociedad-espanola-de-oftalmologia/)